tests/registry_test.go

```diff
@@ -0,0 +1,76 @@
+package tests
+
+import (
+	"context"
+	"math/big"
+	"os"
+	"testing"
+
+	"github.com/klaytn/klaytn/accounts/abi/bind"
+	"github.com/klaytn/klaytn/accounts/abi/bind/backends"
+	"github.com/klaytn/klaytn/blockchain/system"
+	"github.com/klaytn/klaytn/common"
+	"github.com/klaytn/klaytn/consensus/istanbul"
+	"github.com/klaytn/klaytn/contracts/system_contracts"
+	"github.com/klaytn/klaytn/log"
+	"github.com/klaytn/klaytn/params"
+	"github.com/stretchr/testify/assert"
+)
+
+// Test Registry contract installation at Cancun hardfork
+// under the condition similar to the Cypress network.
+func TestRegistryFork(t *testing.T) {
+	log.EnableLogForTest(log.LvlError, log.LvlInfo)
+
+	forkParentNum := big.NewInt(4)
+	forkNum := big.NewInt(5)
+
+	// Start blockchain node
+	config := params.CypressChainConfig.Copy()
+	config.Istanbul.SubGroupSize = 1
+	config.Istanbul.ProposerPolicy = uint64(istanbul.RoundRobin)
+	config.LondonCompatibleBlock = big.NewInt(0)
+	config.IstanbulCompatibleBlock = big.NewInt(0)
+	config.EthTxTypeCompatibleBlock = big.NewInt(0)
+	config.MagmaCompatibleBlock = big.NewInt(0)
+	config.KoreCompatibleBlock = big.NewInt(0)
+	config.ShanghaiCompatibleBlock = big.NewInt(0)
```

Comment:
What about using `common.Big0` instead of big.NewInt(0)?

Reply:
Fixed in 760bd6a.